src/vs/workbench/contrib/terminalContrib/find/browser/terminalFindWidget.ts

```diff
@@ -81,9 +81,9 @@ export class TerminalFindWidget extends SimpleFindWidget {
 	override reveal(): void {
 		const initialInput = this._instance.hasSelection() && !this._instance.selection!.includes('\n') ? this._instance.selection : undefined;
 		const xterm = this._instance.xterm;
-		if (xterm && this.inputValue && this.inputValue !== '') {
+		if (xterm && initialInput && initialInput !== '') {
```

Comment:
do we want to fallback to `this.inputValue` if `initialInput` is undefined?

Reply:
I need to adjust and test it.